src/main.rs

```diff
@@ -179,6 +184,10 @@ struct Opt {
         conflicts_with("panels")
     )]
     terminal_width: Option<NonZeroU64>,
+
+    /// Print a table showing how different types of bytes are colored.
+    #[arg(short('t'), long)]
```

Comment:
Can we remove the short flag here, in case we want to use it for something that is used more frequently in the future?
```suggestion
    #[arg(long)]
```